test/spec/modules/yieldmoBidAdapter_spec.js

```diff
@@ -137,7 +137,7 @@ describe('YieldmoAdapter', () => {
     });
   });
 
-  describe('getUserSync', () => {
+  describe.skip('getUserSync', () => {
```

Comment:
Is this intentionally skipped?

Reply:
Yes it fails in iOS. I have created issue https://github.com/prebid/Prebid.js/issues/2098